src/lib/components/Snackbar/index.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview A Material snackbar for showing notifications.
+ */
+
+import {html} from "lit-element";
+import {BaseElement} from "../BaseElement";
+
+const OPENING_ANIMATION_TIME = 150;
+const CLOSING_ANIMATION_TIME = 75;
+
+class Snackbar extends BaseElement {
+  static get properties() {
+    return {
+      animatable: {type: Boolean, reflect: true},
+      stacked: {type: Boolean, reflect: true},
+      type: {type: String},
+    };
+  }
+
+  get open() {
+    return this.hasAttribute("open");
+  }
+
+  set open(val) {
+    let ms;
+    if (Boolean(val)) {
+      this.setAttribute("open", "");
+      ms = OPENING_ANIMATION_TIME;
+    } else {
+      this.removeAttribute("open");
+      ms = CLOSING_ANIMATION_TIME;
+    }
+
+    this.animatable = true;
+    setTimeout(() => (this.animatable = false), ms);
+  }
+
+  onAccept() {
+    this.open = false;
+    // acceptAction is a property set by an ancestor or an action.
+    if (this.acceptAction) {
+      this.acceptAction();
+      this.acceptAction = null;
+    }
+  }
+
+  get cookiesTemplate() {
+    return html`
+      <div class="web-snackbar__label" role="status" aria-live="polite">
```

Comment:
TIL "polite".

Reply:
Yeah, I think technically it's redundant since I'm using `role=status` and not `role=alert`.

status is implied aria-live=polite
alert is implied aria-live=assertive.

Good reminder that I need to test this with a screen reader before landing it :)

Reply:
I made it so the snackbar is first in the DOM. This is nice because when a user lands on the site they don't immediately have their focus stolen. But it's essentially the first thing they reach with a screen reader.